src/convertToObject.js

```diff
@@ -6,7 +6,16 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map((str) => str.trim())
+    .filter((str) => str.length > 0)
+    .map((str) => str.split(':').map((part) => part.trim()))
+    .reduce((obj, [key, value]) => {
+      obj[key] = value;
+
+      return obj;
+    }, {});
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. This makes the code much more readable. Consider formatting the chaining so that each method call starts on a new line with the same indentation level as the first call in the chain.